app/views/layouts/_footer.html.slim

```diff
@@ -0,0 +1,36 @@
+footer class="govuk-footer no-print" role="contentinfo"
+  div class="govuk-width-container"
```

Comment:
Slim also assumes all elements are divs unless otherwise specified, so you can simply put `.govuk-width-container` for this line